src/components/Admin/Projects/index.js

```diff
@@ -219,6 +251,25 @@ export default class Projects extends React.Component {
             },
           }).then(({data}) => {
             self.props.data.refetch()
+            project.extensions.map(function(projectExtension){
+              if(projectExtension.extension.key === "kubernetesloadbalancers" && projectExtension.environment.id === env.id) {
+                if(projectExtension.customConfig.type === "internal") {
```

Comment:
If we are just updating the load balancers, why does it matter if they are internal vs external?